docs/_configuration/fragmentations/timebased.md

```diff
@@ -29,6 +30,37 @@ For maxGranularity the following values are allowed:
 * minute,
 * second.
 
+### Linear time caching
+This configuration which is off by default makes it possible to activate intelligent caching for time based data.
+When you have a data with timestamps that keep increasing and are always current or newer it possible to improve 
+the caching. **We always assume that future members will have future timestamps.**
+
+**For example**
+timestamp: 21/02/2024
+
+Will result in fragments:
+- root
+- year=2024
+- year=2024&month=02
+- year=2024&month=02&day=21
+
+We will assume that no new year will be added to the root until 31/12/2024.
+We will assume that no new month will be added to year=2024 until 29/02/2024.
```

Comment:
Just to verify, It keeps track of leap years?

Reply:
Yes, we use the default java.time.temporal.TemporalAdjusters to determine the last day of the month. This library keeps track of leap years.